utilities/write_batch_with_index/write_batch_with_index_internal.h

```diff
@@ -97,6 +97,9 @@ class BaseDeltaIterator : public Iterator {
   bool current_at_base_;
   bool equal_keys_;
   bool allow_unprepared_value_;
+  // if set, the iterator will return Entrys with timestamps filled in.
```

Comment:
"Fill timestamps" is a bad name for how to read something. It implies modification.

Also, "will return Entrys with timestamps filled in" - does that mean only entries that already have timestamps or all entries but each with an associated timestamp?

Also, the data model with UDT includes timestamps. Doing something different should not be considered a default or normalized interpretation of that data model. And 'false' are considered default or normal values for bools. How about "ignore_timestamps"?